src/display/editor/tools.js

```diff
@@ -63,8 +63,9 @@ class CommandManager {
    * Add a new couple of commands to be used in case of redo/undo.
    * @param {function} cmd
    * @param {function} undo
+   * @param {boolean} mustExec
    */
-  add(cmd, undo) {
+  add(cmd, undo, mustExec) {
```

Comment:
Please mark the new parameter as optional with `mustExec = false`m and also indicate that in the JSDoc (using `* @param {boolean} [mustExec]`).